packages/usa-date-picker/src/index.js

```diff
@@ -1295,6 +1295,10 @@ const selectDate = (calendarDateEl) => {
  * @param {HTMLButtonElement} el An element within the date picker component
  */
 const toggleCalendar = (el) => {
+  if (el.hasAttribute("aria-disabled")) {
+    el.setAttribute("readonly", true)
+    return;
+  }
   if (el.disabled) return;
```

Comment:
## toggleCalendar function

**issue (blocking)** With the curent approach, the `readonly` attribute is only being added when a user clicks the calendar button.

**suggestion:** We should only prevent calendar functionality here. We can move the `readonly` attributre to the `ariaDisable()` function on line lines [721-726](https://github.com/uswds/uswds/blob/9c3967195abe62643302d0cc91b79e3beba1381f/packages/usa-date-picker/src/index.js#L721-L726)

```suggestion
  if (el.disabled || el.hasAttribute("aria-disabled")) return;
```

Reply:
Resolved in [172c512](https://github.com/uswds/uswds/pull/6013/commits/172c512057f6bf100a7a31171bce0c6878ea56c7). Let me know if this is what was recommended here or if I misunderstood!